lib/util.js

```diff
@@ -1265,6 +1265,7 @@ module.exports = {
     if (operationItem.properties.hasOwnProperty('servers')) {
       serverObj = operationItem.properties.servers[0];
       baseUrl = serverObj.url.replace(/{/g, ':').replace(/}/g, '');
+      baseUrl += reqUrl;
```

Comment:
Also check the change to `pathVarArray` in https://github.com/postmanlabs/openapi-to-postman/pull/93/files